docs/products/kafka/howto/fake-sample-data.rst

```diff
@@ -0,0 +1,63 @@
+Fake sample dataset
+===================
+
+Learning streaming is way more fun with data, so to get you started on your Apache Kafka® journey we help you creating fake streaming data to a topic.
```

Comment:
`we help you creating` doesn't feel quite right - perhaps `we help you create` or `we help you with creating`.